### DISCUSSION TITLE
Staff won't leave rooms

### DISCUSSION THREAD

**Discussion by B17Fan** [CorsixTH/CorsixTH#2299] [General Diagnostics]
I've updated to v.065 and the staff won't leave rooms to help. I've got it ticked in the policy, but they just stand in the rooms, even when there's no patients expected or queued. How do I fix this?

UPDATE: I've now updated to V.066 and I am playing the Crizzman Campaign....The doctor left the gps office and went to General Diagnosis. But the psychiatrist won't leave psychiatry, and the nurse won't leave the ward. 
Is this an issue just for the custom campaign or the original? 

I've not seen this issue flagged anywhere

**Comment by lewri** [comment]
Was this in a new game on 0.65/0.66 or a carryover save from an old version? Ideally we'd need a save that showed the problem as soon as you opened it in the new version.

**Reply by B17Fan** [reply]
I had this on the gamelog a few days ago. 

Created game with savegame version 134.
Warning: Trying to remove nonexistant room build callback (function: 00000142D997AAB0) from humanoid (table: 00000142DA4325B0).
Warning: Trying to remove nonexistant room remove callback (function: 00000142DB7F4EB0) from humanoid (table: 00000142DA4325B0).
Warning: Trying to remove nonexistant staff callback (function: 00000142D997BD80) from humanoid (table: 00000142DA4325B0).
Savegame version is 134 (v0.63), originally it was 134 (v0.63)
Error in timer handler: 
Lua\entities\humanoid.lua:623: Can only finish current action
stack traceback:
	[C]: in function 'assert'
	Lua\entities\humanoid.lua:623: in method 'finishAction'
	Lua\rooms\cardiogram.lua:91: in field 'after_use'
	Lua\humanoid_actions\multi_use_object.lua:316: in local 'timer_function'
	Lua\entity.lua:216: in method 'tick'
	Lua\world.lua:1042: in method 'onTick'
	Lua\app.lua:1111: in function <Lua\app.lua:1108>
	(...tail calls...)
	Lua\app.lua:1012: in function <Lua\app.lua:1007>
